### DISCUSSION TITLE
posts/use-url-class-in-react-native/

### DISCUSSION THREAD

**Discussion by giscus** [dcangulo/davidangulo.xyz#151] [Announcements]
# posts/use-url-class-in-react-native/

If you have been coding React Native for a while you may have noticed that some of Web APIs aren’t implemented in React Native.

https://www.davidangulo.xyz/posts/use-url-class-in-react-native/

**Comment by Rr7716** [comment]
I encounter expo Error: URLSearchParams.set is not implemented when I use expo-cli. So I don't have index.js in my workspace. How to solve this problem？

**Reply by cliffgor** [reply]
You should add it to your App.js and it should work like a charm
